README.md

```diff
@@ -1,6 +1,46 @@
-# Hello World Project
-Simple website build with HTML and CSS
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello World
+
+> Html and CSS website 
+
+## Built With
 
-### Built with
 - HTML
-- CSS 
+- CSS
+
+To get a local copy up and running follow these simple example steps.
+
+### Setup
+- Clone this repo
+
+### Usage
+- Navigate into the cloned repo and open index.html with a browser of your choice
+
+## Authors
+
+👤 **Cecilia Mukima**
+
+- GitHub: [@c3c1l1a](https://github.com/c3c1l1a/)
+- Twitter: [@cMukima](https://twitter.com/CMukima)
+- LinkedIn: [cecilia-wangui-mukima](https://linkedin.com/in/linkedinhandle)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you input your project GitHub issues link into the parenthesis 

### EXAMPLE
`[issues page](https://github.com/c3c1l1a/Hello-world/issues)`